Commands_Functions/create_command.py

```diff
@@ -0,0 +1,30 @@
+import os, json
+from schema_keys import Keys
+from output.exceptions import MissingInput, WrongInput
+from output.status import Status
+
+class CreateCommand():
+    def __init__(self, schema_path):
+        self.schema_data = self.isvalid(schema_path = schema_path)
```

Comment:
Why you didn’t make isValid static ?
IsSomething means it returns a Boolean. I recommend rename it